Specs/Renderer/BufferSpec.js

```diff
@@ -277,6 +281,368 @@ defineSuite([
         expect(buffer.indexDatatype).toEqual(IndexDatatype.UNSIGNED_SHORT);
     });
 
+    it('getBufferData throws without WebGL 2', function() {
```

Comment:
For now, I think this suite should explicitly construct a Cesium context that requests WebGL 2 and use it for these new tests.  A bit longer term, I think CI will need to run all tests with WebGL 2 and WebGL 1 contexts.